public/app/features/dashboard/containers/DashboardPage.tsx

```diff
@@ -378,21 +455,53 @@ export class UnthemedDashboardPage extends PureComponent<Props, State> {
           scrollTop={updateScrollTop}
         >
           <DashboardPrompt dashboard={dashboard} />
-
           {initError && <DashboardFailed />}
           {showSubMenu && (
             <section aria-label={selectors.pages.Dashboard.SubMenu.submenu}>
               <SubMenu dashboard={dashboard} annotations={dashboard.annotations.list} links={dashboard.links} />
             </section>
           )}
-
-          <DashboardGrid
-            dashboard={dashboard}
-            isEditable={!!dashboard.meta.canEdit}
-            viewPanel={viewPanel}
-            editPanel={editPanel}
-          />
-
+          {config.featureToggles.editPanelCSVDragAndDrop ? (
+            <DropZone
+              onDrop={this.onFileDrop}
+              accept={{
+                'text/plain': ['.csv', '.txt'],
```

Comment:
Would also be good to have a constant for accepted file types. Something like DND_ACCEPTED_FILE_TYPES or similar